src/dda/secrets/api.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-FileCopyrightText: 2025-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+import os
+
+import keyring
+
+from dda.config.constants import AppEnvVars
+
+KEYRING_SERVICE = "dda"
+KEYRING_ITEM_API_KEY = "telemetry_api_key"
+KEYRING_ITEM_CLIENT_TOKEN = "feature_flags_client_token"  # noqa: S105 This is not a hardcoded secret but the linter complains on it
```

Comment:
I think documenting every inline exception to rules is untenable (required for global exceptions of course) so perhaps let's remove these if it's all the same to you:
```suggestion
KEYRING_ITEM_CLIENT_TOKEN = "feature_flags_client_token"  # noqa: S105
```